### ISSUE TITLE
Change images to X world

### ISSUE THREAD

**Issue by Santyxs** [Andavin/Images#106]
I would like to know if I can change the images I have in one world to another, for example from world A to world B

**Comment by resi23** [comment]
I don't think this is possible without further ado because the CustomImage stores not only the image but also the name, direction, location and creator in the data.

**Comment by CloudeLecaw** [comment]
see here
https://github.com/Andavin/Images/issues/104

this user also tried something. he renamed thr World and thought he could be manage it with tbat.
And by the way he failed too ";-)
